### ISSUE TITLE
Add distance operators

### ISSUE THREAD

**Issue by NicoLaval** [sdmx-twg/vtl#516]
It is interesting to measure distances between words in statistics.

What about added a group for distance operators, filled by the `levenshtein` distance for now:

```
functions:
    ...
    | distanceOperators                 # distanceFunctions
;

distanceOperators:
    LEVENSHTEIN LPAREN left=expr COMMA right=expr RPAREN    # levenshteinAtom
;
```

It's just an addition, backwards compatibility is ensured.

**Comment by linardian** [comment]
I agree, but since "distance" is also a topic regarding geospatial (and the introduction of geospatial datatype is still on the table), we have to avoid misunderstandings...

**Comment by linardian** [comment]
I agree, but since "distance" is also a topic regarding geospatial (and the introduction of geospatial datatype is still on the table), we have to avoid misunderstandings... @NicoLaval ?

**Comment by NicoLaval** [comment]
Finally, maybe it's simplest to add `levenshtein` function directly to the `stringFunctions` existing branch.
What do you think @linardian?

**Comment by linardian** [comment]
Ok for me.
